### ISSUE TITLE
Keyboard on screen

### ISSUE THREAD

**Issue by mruncleleon** [zerojay/RetroPie-Extra#301]
Hi, is there any keyboard on screen like Steam Lotus Keyboard for Xbox 360 controller? It would be amazing if there were an integrated virtual keyboard for controls rather than a real keyboard connected. Sometimes it is necessary to write in Chromium and the xbox 360 control is not directly supported and I do not find an extension for it in the Google Chrome Web Store.

Example:

![image](https://user-images.githubusercontent.com/39353882/41909133-a3a54cc2-791c-11e8-83db-9169a96de015.png)

**Comment by zerojay** [comment]
Find me a program that we can use that way and I'll see what I can do.

**Comment by symbios24** [comment]
retropie is not android good luck with that

**Comment by mruncleleon** [comment]
> retropie is not android good luck with that

Hi, so far I have been able to use [Chrome-Couch](https://github.com/FroshBite/Chrome-Couch) but I can not type address in Chromium's address bar by restricting Chrome itself. So I found [this Lotus keyboard](https://github.com/likethemammal/daisywheeljs) but I could not try to use it because I'm out of time.
Soon I will post what I did to work on Chromium.
